feat: generate repomap (#41)
trufflehq/truffle-packages: README.md

# truffle-packages

## Repository Map

> More information available in package README

<!-- START PACKAGES -->

**Packages**  
[@truffle/state@`0.0.2`](./state) - Signals coupled with Legend state for React  

> [@truffle/distribute@`2.0.5`](./distribute) - Vendored packages with minor modifications  

Reviewer: ```suggestion
[@truffle/distribute@`2.0.5`](./distribute) - Wrappers to create web components from various frameworks  
```

[@truffle/third-party-oauth@`0.0.31`](./third-party-oauth) - Login and other auth utilities  
[@truffle/events@`0.0.1`](./events) - Utilities for handling webhooks from Truffle  

> [@truffle/config@`1.0.0`](./config) - Default package configuration  

Reviewer: ```suggestion
[@truffle/config@`1.0.0`](./config) - Env/API config  
```

> [@truffle/context@`1.0.0`](./context) - Utilities for context in React  

Reviewer: ```suggestion
[@truffle/context@`1.0.0`](./context) - Framework-agnostic context  
```

> [@truffle/router@`1.0.0`](./router) - Router for Truffle package routes  

Reviewer: ```suggestion
[@truffle/router@`1.0.0`](./router) - Framework-agnostic fs router  
```

> [@truffle/api@`0.1.12`](./api) - Hooks and functions for interacting with Truffle APIs  

Reviewer: ```suggestion
[@truffle/api@`0.1.12`](./api) - Hooks and functions for interacting with Truffle's backend  
```

> [@truffle/ui@`0.1.8`](./ui) - Unified UI  

Reviewer: ```suggestion
[@truffle/ui@`0.1.8`](./ui) - Unified UI library (WIP)  
```

[@truffle/utils@`0.0.21`](./utils) - Browser & Node utilities  
[@truffle/youtube-js@`0.5.9`](./youtube-js) - Truffle fork of [Youtube.js](https://github.com/LuanRT/YouTube.js)  

> [@truffle/global-context@`1.0.0`](./global-context) - Truffle Global Context for React

Reviewer: ```suggestion
[@truffle/global-context@`1.0.0`](./global-context) - Truffle Global Context to share info between packages
```

**Examples**  
[@truffle/events-demo-backend@`0.0.12`](./examples/events-demo-backend) - [Events](../../events) demo  

> [@truffle/create-react-project@`0.5.6`](./examples/create-react-project) - Truffle project template  

Reviewer: ```suggestion
[@truffle/create-react-project@`0.5.6`](./examples/create-react-project) - Truffle project React template  
```
